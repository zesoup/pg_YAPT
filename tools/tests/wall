$config = {
    tests     => 1,
    defaultui => 'default',

    UI => {
        default => {
            template   => "wall",
            updatetime => 10000,    #ns
            checks => [ {check=>"WAL"},{check=> "SIZE"},{check=> "UpTime"}, {check=>"TotRows"}, {check=>"S/I"}, {check=>"I/U/D"} ]
          }

    }
};
