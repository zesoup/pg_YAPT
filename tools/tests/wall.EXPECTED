
[92m  WAL  [0m[93mâ”‚[0m[92m SIZE  [0m[93mâ”‚[0m[92mUpTime [0m[93mâ”‚[0m[92mTotRows[0m[93mâ”‚[0m[92m  S/I  [0m[93mâ”‚[0m[92m I/U/D [0m
[37m 0|0MB [0m[93mâ”‚[0m[37m  0GB  [0m[93mâ”‚[0m[37m  0h   [0m[93mâ”‚[0m[37m 0.0m  [0m[93mâ”‚[0m[37m0/0|0k [0m[93mâ”‚[0m[37m0/0/0|0k[0m
[37m 0|0MB [0m[93mâ”‚[0m[37m  0GB  [0m[93mâ”‚[0m[37m  0h   [0m[93mâ”‚[0m[37m 0.0m  [0m[93mâ”‚[0m[37m0/0|0k [0m[93mâ”‚[0m[37m0/0/0|0k[0m
[37m 0|0MB [0m[93mâ”‚[0m[37m  0GB  [0m[93mâ”‚[0m[37m  0h   [0m[93mâ”‚[0m[37m 0.0m  [0m[93mâ”‚[0m[37m0/0|0k [0m[93mâ”‚[0m[37m0/0/0|0k[0m